### DISCUSSION TITLE
How do you add fonts?

### DISCUSSION THREAD

**Discussion by kharann** [fwouts/previewjs#1359] [Q&A]
I'm trying to add a font, but cant get it working. Currently I used a CSS import, but I cant get it to work

**Comment by fwouts** [comment]
Hey @kharann, CSS import sounds like a good approach but it's possible that your CSS isn't currently being imported by Preview.js.

Check https://previewjs.com/docs/config/css for instructions on setting up a wrapper component which will let you import your CSS.
